crypto/fipsmodule/evp/digestsign.c

```diff
@@ -93,17 +98,26 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
       return 0;
```

Comment:
Let's just, WLOG, assume this point is reached. Who free up the memory `ctx->pctx `? Not really related to this PR, but looks like it leaks.

Reply:
I took a look at `EVP_MD_CTX_free` and noticed it frees all the contents allocated within it: https://github.com/aws/aws-lc/blob/f9f742895d01150e1384d3a4521ccb9b920a7fb3/crypto/fipsmodule/digest/digest.c#L122-L129 

`EVP_MD_CTX` is in charge of freeing the memory within `ctx->pctx`, there's an example of this behavior within this test: 
https://github.com/aws/aws-lc/blob/f9f742895d01150e1384d3a4521ccb9b920a7fb3/crypto/evp_extra/evp_test.cc#L675-L678
`pctx` isn't specifically freed at the end of this test, since the unique pointer`ctx` above will free it.